carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/Screenshot.java

```diff
@@ -24,9 +24,20 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 import javax.imageio.ImageIO;
 
+import com.qaprosoft.carina.core.foundation.utils.messager.ZafiraMessager;
+import com.qaprosoft.zafira.client.ZafiraSingleton;
+import com.qaprosoft.zafira.listener.ZafiraListener;
+import com.qaprosoft.zafira.log.MetaInfoLevel;
+import com.qaprosoft.zafira.log.MetaInfoMessage;
+import com.qaprosoft.zafira.models.dto.TestType;
+import com.qaprosoft.zafira.models.dto.aws.FileUploadType;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.StringUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)